docs/how-to/kubernetes.md

```diff
@@ -351,6 +351,47 @@ matches = queried_docs[0].matches
 print(f"Matched documents: {len(matches)}")
 ```
 
+## Update your Executor in Kubernetes
+
+In Kubernetes, you can update your Executors by patching the deployment corresponding to your Executor.
```

Comment:
"Deployment" or "deployment"? I think either is fine because it is a k8s term, but not only; but we should capitalize it consistently (here it isn't, below it is).